src/config/media.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
please move configs out of `src` dir. The `src` dir in package is the same as `app` dir in the application, so `configs`, migrations`, `resources`, etc. should be placed in the base path

- src
- configs
- migrations
- tests
- composer.json
- .gitignore

Reply:
Done.